pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -489,6 +498,17 @@ func Run() {
 			go FlowStatsCollect(&zedrouterCtx)
 			agentlog.CheckMaxTime(agentName, start)
 
+		case <-HostProbeTimer.C:
+			start := agentlog.StartTime()
+			log.Infof("HostProbeTimer at %v", time.Now())
+			// launch the go function gateway/remote hosts probing check
+			go launchHostProbe(&zedrouterCtx)
```

Comment:
If the probes are slow this construct means you might run multiple concurrent go routines probing; you don't wait for a previous one to finish before starting a new one, do you?

Reply:
No, we don't. i can change this into timer vs ticker